### PR TITLE
ignore yarnclean, and add example for .env uses

### PR DESCRIPTION
**Pull request by dotanthanh**
changes made to project at root folder
- only `.yarnclean` is omitted, for now lock files are kept
- example file for `.env`

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# variable name follow: REACT_APP_<name-of-your-variable>
+REACT_APP_TITLE=React Boilerplate
```
Comment (kernesti) [inline .env.example]: Is this really everything you have in your .env? Where is the port configuration, backend configuration, etc?

File: .gitignore [hunk 2]
```diff
@@ -18,3 +18,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.yarnclean
```
Comment (kernesti) [inline .gitignore]: Aparently the react-boilerplate project also includes these files. Let's not ignore them then even though they are annoying.

File: .env.example (hunk 1 above)
Comment (dotanthanh) [reply .env.example]: added more in .env.example

File: .gitignore (hunk 2 above)
Comment (dotanthanh) [reply .gitignore]: removed

File: .env.example (hunk 1 above)
Comment (kernesti) [reply .env.example]: looks good to me

**Pull request merged**